src/gc/boehm.cr

```diff
@@ -50,7 +50,8 @@ require "crystal/tracing"
 {% end %}
 lib LibGC
   {% unless flag?(:win32) %}
-    VERSION = {{ `pkg-config bdw-gc --silence-errors --modversion || printf "0.0.0"`.chomp.stringify }}
+    {% pkg_config_name = ((ann = LibGC.annotations.find(&.["pkg_config"])) && ann["pkg_config"]) || ((ann = LibGC.annotations.find(&.[0])) && ann[0]) %}
```

Comment:
[This overload is only available in 1.7.0](https://github.com/crystal-lang/crystal/pull/9326) so the annotation type has to be specified (probably a good idea):

```suggestion
    {% pkg_config_name = ((ann = LibGC.annotations(Link).find(&.["pkg_config"])) && ann["pkg_config"]) || ((ann = LibGC.annotations(Link).find(&.[0])) && ann[0]) %}
```